src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10841,6 +10841,28 @@ void getVRFTUZoneLoad(
     }
 }
 
+bool getVRFTUNodeNumber(EnergyPlusData &state, int const nodeNumber)
+{
+    for (int vrfTUIndex = 1; vrfTUIndex <= state.dataHVACVarRefFlow->NumVRFTU; ++vrfTUIndex) {
+        auto &vrfTU = state.dataHVACVarRefFlow->VRFTU(vrfTUIndex);
+
+        bool noVrfTUOutdoorAir = false;
+        if (vrfTU.CoolOutAirVolFlow == 0 && vrfTU.HeatOutAirVolFlow == 0 && vrfTU.NoCoolHeatOutAirVolFlow == 0) {
+            noVrfTUOutdoorAir = true;
+        }
+
+        if (noVrfTUOutdoorAir &&
+            (nodeNumber == vrfTU.VRFTUInletNodeNum || nodeNumber == vrfTU.VRFTUOutletNodeNum || nodeNumber == vrfTU.fanInletNode ||
+             nodeNumber == vrfTU.fanOutletNode || nodeNumber == vrfTU.heatCoilAirOutNode || nodeNumber == vrfTU.coolCoilAirOutNode ||
+             nodeNumber == vrfTU.VRFTUOAMixerOANodeNum || nodeNumber == vrfTU.VRFTUOAMixerRelNodeNum || nodeNumber == vrfTU.VRFTUOAMixerRetNodeNum ||
+             nodeNumber == vrfTU.VRFTUOAMixerMixedNodeNum || nodeNumber == vrfTU.SuppHeatCoilAirInletNode ||
+             nodeNumber == vrfTU.SuppHeatCoilAirOutletNode)) {
+            return true;
+        }
+    }
+    return false;
+}
+
```

Comment:
New function to identify nodes used by VRF terminal units that don't use any OA so they can be excluded from the AFN distribution validation.